ctl/authz/authz.go

```diff
@@ -35,43 +38,122 @@ const (
 
 var log = logger.NewLoggerScope("kmeshctl/authz")
 
+// NewCmd returns the root authz command with its subcommands.
 func NewCmd() *cobra.Command {
-	cmd := &cobra.Command{
+	authzCmd := &cobra.Command{
 		Use:   "authz",
-		Short: "Enable or disable xdp authz eBPF Prog for Kmesh's authz offloading",
-		Example: `# Enable/Disable Kmesh's authz offloading in the specified kmesh daemon:
- kmeshctl authz <kmesh-daemon-pod> enable/disable
- 
- # If you want to enable or disable authz offloading of all Kmeshs in the cluster
- kmeshctl authz enable/disable`,
-		Args: cobra.MinimumNArgs(1),
+		Short: "Manage xdp authz eBPF program for Kmesh's authz offloading",
+	}
+
+	authzCmd.AddCommand(NewEnableCmd())
+	authzCmd.AddCommand(NewDisableCmd())
+	authzCmd.AddCommand(NewStatusCmd())
+
+	return authzCmd
+}
+
+// NewEnableCmd creates a command to enable authz.
+func NewEnableCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "enable [podNames...]",
+		Short:   "Enable xdp authz eBPF program for Kmesh's authz offloading",
+		Example: "kmeshctl authz enable\nkmeshctl authz enable pod1 pod2",
+		Args:    cobra.ArbitraryArgs,
 		Run: func(cmd *cobra.Command, args []string) {
-			SetAuthz(cmd, args)
+			// If no pod names are given, apply to all kmesh daemon pods.
```

Comment:
```suggestion
			// If no pod names are given, apply to all Kmesh daemon pods.
```

Please use `Kmesh` uniformly, with the first letter capitalized

Reply:
yup i will take care next time